cert-manager/base/openshift-ingress-issuer.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: ingress-letsencrypt-production
+  namespace: openshift-ingress
+spec:
+  acme:
```

Comment:
when I build the balrog ovelray, the `email` `server` and `privateKeySecretRef` don't seem to get consumed, I get: 

```yaml
apiVersion: cert-manager.io/v1
kind: Issuer
metadata:
  name: ingress-letsencrypt-production
  namespace: openshift-ingress
spec:
  acme:
    solvers:
    - dns01:
        cnameStrategy: Follow
        route53:
          accessKeyID: <redacted>
          region: eu-west-2
          secretAccessKeySecretRef:
            key: secret-access-key
            name: route53
      selector:
        dnsZones:
        - aws.operate-first.cloud


```

Reply:
You're right, seems the ksops "merge" annotation doesn't work as expected :slightly_frowning_face:  I'll see what I can do.

Reply:
I've removed it from ksops. we expose the access key id now. :shrug: